identify/README.md

```diff
@@ -109,7 +109,9 @@ in [peer-ids](../peer-ids).
 
 ### listenAddrs
 
-These are the addresses on which the peer is listening as multi-addresses.
+The addresses on which the peer is deemed to be reachable by other peers.
+These are also sometimes referred to as _external_ addresses.
+For backwards-compatibility reasons, implementations MAY also include addresses of local interfaces.
```

Comment:
There's no value in including local addresses.
```suggestion
```

Reply:
I assume what you meant to say is that peers need to be able to handle local interface addresses that are sent here. But that's true anyway, you always need to be able to handle nonsense that a peer sends you.